eng/pipelines/msquic.yml

```
@@ -58,10 +58,10 @@ stages:
         isOfficialBuild: true
         pool:
           name: NetCore1ESPool-Internal
-          demands: ImageOverride -equals Build.Windows.10.Amd64.VS2019
+          demands: ImageOverride -equals '1es-windows-2019'
       ${{ if or(eq(variables['System.TeamProject'], 'public'), in(variables['Build.Reason'], 'PullRequest')) }}:
         pool:
-          vmImage: 'windows-latest'
+          vmImage: '1es-windows-2019-open'
```

```suggestion
          name: NetCore-Public
          demands: '1es-windows-2019-open'
```

Sorry I missed that you were doing azdo hosted before.  Feel free to just keep doing azdo hosted as well